lib/screens/authentication/import_key/import_words_screen.dart

```diff
@@ -62,9 +62,10 @@ class ImportWordsScreen extends StatelessWidget {
                                 left: (index % _numberOfColumns == 0) ? 0 : 8,
                                 right: ((index + 1) % _numberOfColumns == 0) ? 0 : 8),
                             child: Autocomplete<String>(
-                              fieldViewBuilder: (BuildContext context, TextEditingController textEditingController,
-                                  FocusNode focusNode, VoidCallback onFieldSubmitted) {
+                              fieldViewBuilder: (context, textEditingController, focusNode, onFieldSubmitted) {
                                 textEditingController.text = state.userEnteredWords[index] ?? "";
+                                textEditingController.selection =
```

Comment:
i could swear i added this code in there... i saw this issue happening when i was developing.